### DISCUSSION TITLE
Ability to set the Ding Timeout. It's currently 20 seconds.

### DISCUSSION THREAD

**Discussion by 4rianton** [tsightler/ring-mqtt#887] [Feature Requests]
Thanks for this great add on. I use a Ring Intercom and I had an idea of creating a pattern of presses that will trigger opening the lock. 

Currently it's set to 20000ms but I would like to be able to set it (to 1 second in my case). Is it possible? 

<img width="620" alt="Screenshot 2024-08-12 at 13 50 36" src="https://github.com/user-attachments/assets/68f08f4e-5588-471b-8468-afc26a2679c0">

**Comment by tsightler** [comment]
You seem to think that this is a button press event, but this is not, this is a "ding" notification, i.e. a push notification from Ring that someone has pressed the button.  Ring does not send a new ding notification every time the button is pushed as that would make little sense.  Imagine someone walking up to your door and pressing the button 100 times and you receiving 100 push notifications on your phone.

The Ring intercom is not a programmable button, the processing of button presses happens locally, then a notification is sent.  Ring could potentially implement a feature that allowed you to program different notification types based on double-press, long-press, etc., but I doubt they will ever do that.  Regardless, it's not something that could be done here, regardless of this timeout as the cooldown period for the next notification is significantly longer than 20 seconds.

**Reply by 4rianton** [reply]
Ah! I got it. Makes sense that Ring has no need to expose this as a raw button press in their API. 

Thank you for looking into it, your explanation, and this add-on.🙏
